### PR TITLE
Bump select2 version up

### PR DESCRIPTION
**Pull request by imbyungjun**
* Bump select2 version up to 4.0.13
* Update select2 related codes to fit in the new version
* Remove unnecessary library files.

### CODE REVIEW

File: ngrinder-frontend/src/js/components/perftest/detail/Config.vue [hunk 1]
```diff
@@ -409,8 +410,15 @@
...
+            const scriptName = this.test.config.scriptName;
+            const scriptRevision = this.test.config.scriptRevision;
             this.syncGitHubConfigRevision();
-            this.setScripts(this.test.config.scriptName);
+
+            this.test.config.scriptName = scriptName;
+            this.setScripts(scriptName);
+            this.$nextTick(() => {
+                this.test.config.scriptRevision = scriptRevision;
+            });
```
Comment (imbyungjun) [inline ngrinder-frontend/src/js/components/perftest/detail/Config.vue:421]: There is a change about `change` event on Select2 v4.x.
In prior version, we could use `input` and `change` event seperately. So, there's a benefit to handling a value selection and handling a change of value(by clicking).
But, in the new version, we must trigger the `change` event to apply the change of value appropriately.

The code change above is for fixing select value lost by `input` -> `change` event trigger sequence.

File: ngrinder-frontend/src/js/components/perftest/detail/Detail.vue [hunk 2]
```diff
@@ -160,7 +163,7 @@
                 progressMessage: test.progressMessage,
                 lastProgressMessage: test.lastProgressMessage,
                 testName: test.testName,
-                tagString: test.tagString,
+                tagList: test.tagString !== '' ? test.tagString.split(',') : [],
```
Comment (donggyu04) [inline ngrinder-frontend/src/js/components/perftest/detail/Detail.vue:166]: `!== ''` seems unnecessary code 

`test.tagString ? test.tagString.split(',') : []`

**Review by donggyu04 — DISMISSED** [review]

**Review by junoyoon — DISMISSED** [review]

File: ngrinder-frontend/src/js/components/perftest/detail/Detail.vue (hunk 2 above)
Comment (imbyungjun) [reply ngrinder-frontend/src/js/components/perftest/detail/Detail.vue:166]: Amended it. Thanks!

File: ngrinder-frontend/src/js/components/perftest/detail/Config.vue [hunk 3]
```diff
@@ -137,7 +137,7 @@
                              @change="changeScript"
                              @opening="openingScriptSelect"
                              :validationRules="{ required: true, scriptValidation: true }" errStyle="position: absolute; padding-left: 177px;">
-                        <option></option>
+                        <option data-validate="-1"></option>
```
Comment (imbyungjun) [inline ngrinder-frontend/src/js/components/perftest/detail/Config.vue:140]: This change prevents to `refreshTargetHosts` API call when the script option is empty.

**Review by donggyu04 — APPROVED** [review]

**Pull request merged**